It seems there is a problem with the `otl` library in the failed checks:
```
-- Using cached C:/vsts/_work/4/s/downloads/otlv4_40451.zip
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:99 (message):
  

  File does not have expected hash:

          File path: [ C:/vsts/_work/4/s/downloads/otlv4_40451.zip ]
      Expected hash: [ add1e54fae20175461d5f09cbe324e98b6f6a3839356136811109cf3251ca96541c101816870c6cc15d7611ad6bf9d576ac8dfce4274419b30866955c5892d15 ]
        Actual hash: [ 74499b79a756c1ecc16d6bd9b277b91add55249d2e1cb65a31786e9750ff607da1222797f318954599efc5153cecee2d3a7fd1e994c09e7ddb2507c8e5ce5c55 ]

  Please delete the file and retry if this file should be downloaded again.

Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:110 (test_hash)
  ports/otl/portfile.cmake:3 (vcpkg_download_distfile)
  scripts/ports.cmake:90 (include)
```
It doesn't looks like libzippp related changes are implied here :thinking:

--- a/ports/libzippp/CONTROL
+++ b/ports/libzippp/CONTROL
@@ -1,4 +1,5 @@
 Source: libzippp
-Version: 2019-07-22
+Version: 3.1-1.6.1
+Homepage: https://github.com/ctabin/libzippp
 Description: Simple basic C++ wrapper around the libzip library. It is meant to be a portable and easy-to-use library for ZIP handling
 Build-Depends: zlib, libzip[bzip2]

--- a/ports/libzippp/portfile.cmake
+++ b/ports/libzippp/portfile.cmake
@@ -1,25 +1,28 @@
-include(vcpkg_common_functions)
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO ctabin/libzippp
-    REF be75a3413b648a3264e94a2c1921c83081dec1e0
-    SHA512 86c6040bbaea0817486218e96c4d230a328e3560ada41861fbd18d78faa085b158199318d633085e616509084082bf29d6f97afdd2d2dfbc6b843dfbf6a20c85
-    HEAD_REF master
+    REF fb07ca80ebda0576366619b87364b4b3a94426df
+    SHA512 c75518d0dba43ca5ae617569c0e3105af4f903a1226bdae100ee0770b4745ce4c889d29788bd2d75e7c8b556a5c339c0610170a620081d67a86cf6639e9e7fb2
+    HEAD_REF libzippp-v3.1-1.6.1 
 )
 
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
     OPTIONS
-        -DBUILD_TESTS=OFF
+        -DLIBZIPPP_BUILD_TESTS=OFF
     OPTIONS_DEBUG
-        -DINSTALL_HEADERS=OFF
+        -DLIBZIPPP_INSTALL_HEADERS=OFF
 )
 
 vcpkg_install_cmake()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH "share/libzippp")
+if(WIN32)
+    vcpkg_fixup_cmake_targets(CONFIG_PATH "cmake/libzippp")
+else()
+    vcpkg_fixup_cmake_targets(CONFIG_PATH "share/libzippp")
+endif()
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
 

